CONTRIBUTING.md

```diff
@@ -31,52 +37,60 @@ After the dependencies were installed and the VM initialized, you can proceed wo
 8. Link the PR to the relevant issue
 9. Send the PR to the slack group and ask for reviewers(one student and one mentor at least)
 10. Ensure that your code follows the PEP 8 Style guide for Python code before submitting a pull request.
-      We have also set up Flake8 to ensure that our style guides are being followed(PR's that don't pass this may not get reviewed until those are solved)
+    We have also set up Flake8 to ensure that our style guides are being followed(PR's that don't pass this may not get reviewed until those are solved)
 11. Make sure the PR passes CI checks.
 
-# Pull Request Standards 
- - Each Pull Request should focus on single responsibility principle.
- - **The Pull Request should not break any of the existing functionality**
- - <ins>Description-</ins> the PR description should explain what changes have been made, and why. In addition, they should include a link to the relevant `issue`
- - Each pull request requires the approval of at least <ins>1 team members</ins> and one mentor before merging.
+# Pull Request Standards
+
+- Each Pull Request should focus on single responsibility principle.
+- **The Pull Request should not break any of the existing functionality**
+- <ins>Description-</ins> the PR description should explain what changes have been made, and why. In addition, they should include a link to the relevant `issue`
+- Each pull request requires the approval of at least <ins>1 team members</ins> and one mentor before merging.
+- The Pull Requests title should be in the format of: <Component name>: explanation in a few words
```

Comment:
This is the only real change all the other changes are my formatter arranging the md file spaces and readability